.github/workflows/ci-js.yml

```diff
@@ -0,0 +1,78 @@
+name: CI JS
```

Comment:
Is this 100% a copy of `ci-go` with the npm section and the `create-turbo` tests added?

Reply:
pretty much, but excludes the [`examples`][1] action (terminology?) and excludes [the Go tests][2]

[1]: https://github.com/vercel/turborepo/blob/44ecee64c3c0060f5f7317c5f297b5eb24e09887/.github/workflows/ci-go.yml#L76

[2]: https://github.com/vercel/turborepo/blob/44ecee64c3c0060f5f7317c5f297b5eb24e09887/.github/workflows/ci-go.yml#L70-L74